app/views/guidance/apply-guidance-for-farmers.njk

```diff
@@ -0,0 +1,319 @@
+
+      <p class="govuk-body">Ask the vet for a summary of the review that you can use as evidence. The summary must be a separate document. It can be a digital or paper version.</p>
+
+      <p class="govuk-body">The vet’s summary must include:</p>
+
+      <ul class="govuk-list govuk-list--bullet">
+        <li>the agreement number</li>
+        <li>the business name and <abbr title="single business identifier">SBI</abbr> number where the review took place</li>
+        <li>the date of the vet’s last visit to the farm for the review</li>
+        <li>the type of livestock reviewed</li>
+        <li>confirmation the minimum number of animals being reviewed were on the farm when the vet did the review</li>
+        <li>confirmation the vet carried out testing in accordance with the review’s disease and endemic conditions testing guidance</li>
+        <li>the date the vet completed disease and endemic condition testing</li>
+        <li>the unique reference number for laboratory test results</li>
+        <li>the vet’s name and Royal College of Veterinary Surgeons (<abbr title="Royal College of Veterinary Surgeons">RCVS</abbr>) number</li>
+        <li>confirmation the vet has given you a written report</li>
+    </ul>
+
+    <p class="govuk-body">You must ask the vet to sign and date the summary. The signature can be digital or on paper.</p>
+
+    <h2 id="apply-for-a-review" class="govuk-heading-m">Apply for a review</h2>
+
+    <section class="gem-c-devolved-nations">
+      <h2 class="govuk-heading-s govuk-!-margin-bottom-0">
+        <p class="govuk-body">Find out <a rel="external" href="/apply/start" class="govuk-link">how to apply for an annual health and welfare review of livestock.</a></p>
+      </h2>
+    </section>
+
+    <h2 id="how-to-claim-funding" class="govuk-heading-m">How to claim your funding</h2>
+
+    <p class="govuk-body">When the vet has completed the review, you can claim your funding.</p>
+
+    <p class="govuk-body">Read guidance on how to claim for an annual health and welfare review of your livestock.</p>
+
+    <h2 id="get-help-with-your-application" class="govuk-heading-m">Get help with your application</h2>
+  
+      <p class="govuk-body">Contact the <abbr title="Rural Payments Agency">RPA</abbr> if you:</p>
+  
+      <ul class="govuk-list govuk-list--bullet">
+        <li>decide you do not want a review after you have accepted the agreement offer</li>
+        <li>cannot apply online or have a question about your application</li>
+      </ul>
+  
+      <p class="govuk-body">Have your agreement number with you when you call or include it in your letter or email.</p>
+
+      <div class="govuk-inset-text">
+        <p class="govuk-body">Rural Payments Agency<br>
+        PO Box 352<br>
+        Worksop<br>
+        S80 9FG<br>
+        </p>
+
+        {{ rpaContactDetails({email: ruralPaymentsAgency.email, callChargesUri: ruralPaymentsAgency.callChargesUri, telephone: ruralPaymentsAgency.telephone}) }}
```

Comment:
There is an additional blank line prior to the "Find out about call charges" link within the build that isn't in the prototype